bundles/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/internal/handler/KeContactHandler.java

```diff
@@ -281,12 +281,17 @@ protected void onData(ByteBuffer byteBuffer) {
                         maxSystemCurrent = state;
                         State newState = new DecimalType(state);
                         updateState(new ChannelUID(getThing().getUID(), CHANNEL_MAX_SYSTEM_CURRENT), newState);
-                        if (maxSystemCurrent < maxPresetCurrent) {
-                            transceiver.send("curr " + String.valueOf(maxSystemCurrent), this);
-                            updateState(new ChannelUID(getThing().getUID(), CHANNEL_MAX_PRESET_CURRENT),
-                                    new DecimalType(maxSystemCurrent));
-                            updateState(new ChannelUID(getThing().getUID(), CHANNEL_MAX_PRESET_CURRENT_RANGE),
-                                    new PercentType((maxSystemCurrent - 6000) * 100 / (maxSystemCurrent - 6000)));
+                        if (maxSystemCurrent != 0){
+                            if (maxSystemCurrent < maxPresetCurrent) {
+                                transceiver.send("curr " + String.valueOf(maxSystemCurrent), this);
+                                updateState(new ChannelUID(getThing().getUID(), CHANNEL_MAX_PRESET_CURRENT),
+                                        new DecimalType(maxSystemCurrent));
+                                updateState(new ChannelUID(getThing().getUID(), CHANNEL_MAX_PRESET_CURRENT_RANGE),
+                                        new PercentType((maxSystemCurrent - 6000) * 100 / (maxSystemCurrent - 6000)));
+                            }
+                        }
```

Comment:
remove the line break

Reply:
sorry i did not get it, what linebreaks needs to be removed? from my view here it look good.

Reply:
The else should be on the same line as the closing bracket:
```
} else {
```